hack/gardener-extensions-down.sh

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/env bash
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+SCRIPT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )"
+
+SEED_NAME=""
+PATH_SEED_KUBECONFIG=""
+PATH_GARDEN_KUBECONFIG=""
+
+parse_flags() {
+  while test $# -gt 0; do
+    case "$1" in
+    --seed-name)
+      shift; SEED_NAME="$1"
+      ;;
+    --path-seed-kubeconfig)
+      shift; PATH_SEED_KUBECONFIG="$1"
+      ;;
+    --path-garden-kubeconfig)
+      shift; PATH_GARDEN_KUBECONFIG="$1"
+      ;;
+    esac
+
+    shift
+  done
+}
+
+parse_flags "$@"
+
+# delete stuff gradually in the right order, otherwise several dependencies will prevent the cleanup from succeeding
+echo "Deleting all shoots"
+# Deleting all shoots to ensure that there are no orphan infrastructure elements left
+kubectl --kubeconfig="$PATH_GARDEN_KUBECONFIG" annotate shoots -A --all confirmation.gardener.cloud/deletion=true --overwrite
+kubectl --kubeconfig="$PATH_GARDEN_KUBECONFIG" delete shoots -A --all --wait
```

Comment:
I would remove this, let's rather keep this in the responsibility of the user. Instead, check if there are still shoots and abort with an error.

Reply:
I removed these two statements. 
The check for shoots on the seed is carried out anyway when you try to delete the seed as it is done in the next line.